tools/purge_non_existing_images.sh

```diff
@@ -1,29 +1,52 @@
 #!/bin/sh
 
+DRYRUN=no
+
+if [ "$1" = "-p" ]; then
```

Comment:
I left it out of my comment, but DRYRUN also implies that the logic needs to be inversed, else DRYRUN=no implies that it should purge, but the code will actually imply that it should NOT purge

DRYRUN == dry run, i.e. empty run, i.e. with no changes.